tests/unit/test_client_logging.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from unittest import mock
+
+from google.api_core.client_logging import (
+    setup_logging,
+    initialize_logging,
+)
+
+
+def reset_logger(scope):
+    logger = logging.getLogger(scope)
+    logger.handlers = []
+    logger.setLevel(logging.NOTSET)
+    logger.propagate = True
+
+
+def test_setup_logging_w_no_scopes():
+    with mock.patch("google.api_core.client_logging._BASE_LOGGER_NAME", "foo"):
+        setup_logging()
+        base_logger = logging.getLogger("foo")
+        assert base_logger.handlers == []
+        assert not base_logger.propagate
+        assert base_logger.level == logging.NOTSET
```

Comment:
We should be consistent with the other test cases as to whether we have the `getLogger` and the assertions within the `with` statement.

```suggestion
    with mock.patch("google.api_core.client_logging._BASE_LOGGER_NAME", "foo"):
        setup_logging()
    base_logger = logging.getLogger("foo")
    assert base_logger.handlers == []
    assert not base_logger.propagate
    assert base_logger.level == logging.NOTSET
```